src/main/java/duke/dukeexceptions/DukeExceptions.java

```diff
@@ -0,0 +1,11 @@
+package duke.dukeexceptions;
```

Comment:
I suggest the naming of the package is duke.exceptions

Reply:
Duke exception is a specialised exception for duke so I actually prefer to name it that way.